crates/fluvio-sc-schema/src/objects/watch.rs

```diff
@@ -26,7 +26,7 @@ pub struct WatchRequest<S: AdminSpec> {
 
 impl Request for ObjectApiWatchRequest {
     const API_KEY: u16 = AdminPublicApiKey::Watch as u16;
-    const DEFAULT_API_VERSION: i16 = 6;
+    const DEFAULT_API_VERSION: i16 = 8;
```

Comment:
I don't love this but don't know what it should be. Do we have a global const for these?

Reply:
They are independent versions so they should not be global.